src/control/system/raft/database_test.go

```diff
@@ -431,6 +431,51 @@ func ignoreFaultDomainIDOption() cmp.Option {
 		}, cmp.Ignore())
 }
 
+func checkMemberDB(t *testing.T, mdb *MemberDatabase, expMember *Member, opts ...cmp.Option) {
```

Comment:
This could be exposed as part of a `daos_server ms check` command. An optional `--repair` flag would then attempt to fix any inconsistencies found. On the one hand, repairing silently is potentially more convenient, but on the other hand, there may be other serious problems that are hidden until they're not.